packages/common/lib/common/tools.js

```diff
@@ -1574,10 +1574,11 @@ async function uninstallNodeModule(packageName, options = {}) {
 /**
  * Rebuilds all native node_modules that are dependencies of the project in the current working directory / project root.
  * If `options.cwd` is given, the directory must contain a lockfile.
+ * @param {string|undefined} module module to rebuild e.g. bluetooth-hci-socket@0.5.3-8
  * @param {RebuildNodeModulesOptions} options Options for the rebuild
```

Comment:
Please make `module` an optional property of RebuildNodeModulesOptions. I chose the "bag-of-options" pattern on purpose to avoid the overload hell we have elsewhere.